pageserver/src/tenant/mgr.rs

```diff
@@ -1517,21 +1521,146 @@ impl TenantManager {
             }
         }
 
-        // Phase 5: Shut down the parent shard.
+        // Phase 5: Shut down the parent shard, and erase it from disk
         let (_guard, progress) = completion::channel();
         match parent.shutdown(progress, false).await {
             Ok(()) => {}
             Err(other) => {
                 other.wait().await;
             }
         }
+        let local_tenant_directory = self.conf.tenant_path(&tenant_shard_id);
+        let tmp_path = safe_rename_tenant_dir(&local_tenant_directory)
+            .await
+            .with_context(|| format!("local tenant directory {local_tenant_directory:?} rename"))?;
+        task_mgr::spawn(
```

Comment:
This is duplicative wrt the detach_tenant code: this can be cleaned up when we finish refactoring all the manager stuff into TenantManager (currently detach is still standalone functions)

Reply:
Should it be a TODO comment? Need to check how does `None, None` work with shutdown, I think I last had them as the last tasks to be shut down.

Reply:
The clean up is a small change: opened PR for the followup here: https://github.com/neondatabase/neon/pull/6732

The None,None part is the pre-existing way that we spawn the tasks for deleting during detach: the order during shutdown shouldn't matter because we're deleting a tmpdir with a random prefix